### ISSUE TITLE
How to training for video object segmentation

### ISSUE THREAD

**Issue by limoran233** [alanlukezic/d3s#19]
hello，thanks for your excellent job! I want to know whether the code can be trained on YouTube-VOS and then evaluate on DAVIS for VOS task，and how to implement it？Looking forward to your reply.

**Comment by alanlukezic** [comment]
The pre-trained model which was used for reporting results in the paper and is available online has already been trained on YT-VOS. If you want to use it on DAVIS you have to create your own script which initializes the tracker on the first frame and then tracks it on every other frame. Current implementation can save the predicted segmentation masks if params.masks_save_path in the pytracking/parameter/segm/default_params.py is set to the directory where you want the masks to be saved.
